File: Week 1 — C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for height and validate input
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1);

    for (int i = 0; i < n; i++)
    {
        // Left pyramid (print spaces before hashes)
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        // Left pyramid (print hashes)
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        // Print the double-spaces
        printf("  ");
        // Right pyramid (print hashes)
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        // Move to next line
        printf("\n");
    }
}

File: Week 06 — Python/sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    try:
        height = get_int("Height: ")
        if 1 <= height <= 8:
            break
    except ValueError:
        pass

for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i)
